api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -144,18 +149,15 @@ enum BlobStatus {
   // ENCODED means that the blob has been Reed-Solomon encoded into chunks and is ready to be dispersed to DA Nodes.
   ENCODED = 2;
 
-  // CERTIFIED means the blob has been dispersed and attested by the DA nodes.
-  CERTIFIED = 3;
+  // GATHERING_SIGNATURES means that the blob chunks have been transmitted to validators and is waiting for the attestations from DA network.
```

Comment:
```suggestion
  // GATHERING_SIGNATURES means that the blob chunks are currently actively being transmitted to validators, requesting for signatures. Requests that timeout or receive errors are resubmitted to DA nodes for some period of time set by the disperser, after which the BlobStatus becomes COMPLETE.
```

Reply:
I guess this description is not the current implementation, but do we still want to have it this way?

Reply:
Actually prob best to explicitly mention that this status is not currently implemented?